config/v3_4_experimental/types/config.go

```diff
@@ -25,3 +31,50 @@ var (
 		PreRelease: "experimental",
 	}
 )
+
+func (cfg Config) Validate(c path.ContextPath) (r report.Report) {
+	systemdPath := "/etc/systemd/system/"
+	units := make(map[string]string)
+
+	for _, unit := range cfg.Systemd.Units {
+		if unit.Dropins == nil {
+			units[unit.Name] = systemdPath + unit.Name
+		} else {
+			for _, dropin := range unit.Dropins {
+				units[dropin.Name] = systemdPath + unit.Name + ".d/" + dropin.Name
```

Comment:
If `a.service` and `b.service` both have a dropin `dropin.conf`, only one of them will end up in `units`.

Also, we don't actually create a dropin unless its `contents` field is specified, so we should condition on that.

Reply:
This will be resolved by changing `units` to a slice of strings.